CRM/Report/Form.php

```diff
@@ -814,6 +814,10 @@ public function setDefaultValues($freeze = TRUE) {
                 $this->_defaults["{$fieldName}_relative"] = $field['default'];
               }
             }
+            elseif (CRM_Utils_Array::value('type', $field) & CRM_Utils_Type::T_INT && is_array($field['default'])) {
```

Comment:
Reason why the related test build is failing and it will not set default values for integer filter other then range kind ($this->_defaults["{$fieldName}_value"]) is because in if condition its only checking is_array() and under the IF scope it only set max/min not _value filter like in else part. So my suggestion would be to handle the same under ELSE  and 
..
else {
 if (CRM_Utils_Array::value('type', $field) & CRM_Utils_Type::T_INT &&  CRM_Utils_Array::value('min', $field['default'])) {
  $this->_defaults["{$fieldName}_min"] = CRM_Utils_Array::value('min', $field['default']);
  $this->_defaults["{$fieldName}_max"] = CRM_Utils_Array::value('max', $field['default']);
 }
}